includes/Modules/Reader_Revenue_Manager/Admin_Post_List.php

```diff
@@ -0,0 +1,241 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Reader_Revenue_Manager\Admin_Post_List
+ *
+ * @package   Google\Site_Kit\Modules\Reader_Revenue_Manager
+ * @copyright 2025 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Reader_Revenue_Manager;
+
+/**
+ * Class for adding RRM elements to the WP Admin post list.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Admin_Post_List {
+
+	/**
+	 * Post_Product_ID instance.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @var Post_Product_ID
+	 */
+	private $post_product_id;
+
+	/**
+	 * Settings instance.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @var Settings
+	 */
+	private $settings;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param Settings        $settings        Module settings instance.
+	 * @param Post_Product_ID $post_product_id Post Product ID.
+	 */
+	public function __construct( Settings $settings, $post_product_id ) {
+		$this->settings        = $settings;
+		$this->post_product_id = $post_product_id;
+	}
+
+	/**
+	 * Registers functionality through WordPress hooks.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function register() {
+		$post_types = $this->get_post_types();
+
+		foreach ( $post_types as $post_type ) {
+			add_filter(
+				"manage_{$post_type}_posts_columns",
+				array( $this, 'add_column' )
+			);
+
+			add_action(
+				"manage_{$post_type}_posts_custom_column",
+				array( $this, 'fill_column' ),
+				10,
+				2
+			);
+		}
+
+		add_action(
+			'bulk_edit_custom_box',
+			array( $this, 'bulk_edit_field' ),
+			10,
+			2
+		);
+
+		add_action( 'save_post', array( $this, 'save_field' ) );
+	}
+
+	/**
+	 * Adds a custom column to the post list.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $columns Columns.
+	 * @return array Modified columns.
+	 */
+	public function add_column( $columns ) {
+		$columns['rrm_product_id'] = __( 'Show Reader Revenue CTAs', 'google-site-kit' );
+		return $columns;
+	}
+
+	/**
+	 * Fills the custom column with data.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $column  Column name.
+	 * @param int    $post_id Post ID.
+	 */
+	public function fill_column( $column, $post_id ) {
+		if ( 'rrm_product_id' !== $column ) {
+			return;
+		}
+
+		$post_product_id = $this->post_product_id->get( $post_id );
+
+		if ( 'none' === $post_product_id ) {
+			esc_html_e( 'None', 'google-site-kit' );
+			return;
+		}
+
+		if ( ! empty( $post_product_id ) ) {
+			if ( 'openaccess' === $post_product_id ) {
+				esc_html_e( 'Open access', 'google-site-kit' );
+			} else {
+				$separator_index = strpos( $post_product_id, ':' );
+				$product_id      = ( false !== $separator_index ) ? substr( $post_product_id, $separator_index + 1 ) : $post_product_id;
+
+				/* translators: %s: Product ID */
+				echo esc_html( sprintf( __( 'Use "%s"', 'google-site-kit' ), $product_id ) );
```

Comment:
According to the ACs, the value here should solely use the product ID, without "_Use_".